riff-raff/public/docs/howto/configure-a-project.md

```diff
@@ -0,0 +1,42 @@
+Configuring a project
+=====================
+
+To make a project deployable with riff-raff it needs:
+ 
+ - A [`riff-raff.yaml`](../reference/riff-raff.yaml.md) file that describes the deployment process
+ - A [`build.json`](../reference/build.json.md) file that details the CI build
+ - The two files above along with any assets uploaded into riff-raff's S3 artifact buckets
+
+Both the [sbt-riffraff-artifact plugin](https://github.com/guardian/sbt-riffraff-artifact) or
+[node-riffraff-artifact plugin](https://github.com/guardian/node-riffraff-artefact) will both create the `build.json`
+for you and help you to upload the files to the S3 buckets correctly.
+
+Using SBT
+---------
+
+ 1. Add [sbt-riffraff-artifact plugin](https://github.com/guardian/sbt-riffraff-artifact) to your `project/plugins.sbt`
```

Comment:
We should probably move these docs into the repo - could probably do with some work. I've tweaked the copy a little, but will leave some of the instructions here for now.